articles/how-to-send-sms-in-django-using-twilio/index.md

```diff
@@ -0,0 +1,230 @@
+- [Creating a model](#creating-a-model)
+- [Configuration](#configuration)
+- [Conclusion](#conclusion)
+
+
+
+### Creating and setting up a new Django project
+Let's start by creating a new virtual environment.
+
+A virtual environment allows you to create different spaces on your computer, with a different set of libraries and versions.
+
+By creating a virtual environment, you'll be able to separate the necessary library installation for a project, without having to install them globally.
+
+Now, you create a virtual environment `env` as shown below:
+
+```bash
+$ python -m venv /path/to/new/virtual/environment
+```
+
+Here, we specify the `/path/to/new/virtual/environment` as `env`.
+
+On creation, you can activate the virtual environment using the following command:
+
+```bash
+$ source env/bin/activate
+```
+
+On activating the environment, we can install Django using the following command:
+
+```bash
+$ pip install django
+```
+
+Now, let's create a new Django project `twilio_proj` using:
+
+```bash
+$ django-admin startproject twilio_proj
+```
+
+Then, we create a Django app.
+
+```bash
+$ django-admin startapp score
+```
+
+Let's add our app to the list of installed apps. Navigate to the `twilio_proj` directory and edit the `settings.py` file.
+
+```python
+INSTALLED_APPS = [
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+    'score',
+    ]
+```
+
+### Setting up Twilio
+To get started using Twilio, we need to sign up on [Twilio's](https://twilio.com) website to create an account and also get a twilio number.
+
+Click on the `Sign Up` button and fill in your details to create an account.
+
+
+### Creating a model
+Now lets define a very simple class.
+
+In the `score` directory, edit the `models.py` file and add the following lines of code to it:
+
+```python
+from django.db import models
+
+#defining a simple class
+class Score(models.Model):
+    #integer field
+    test_result = models.PositiveIntegerField()
+
+    #string representation
+    def __str__(self):
+        return str(self.test_result)
+
+```
+
+Now let's register the model of Score in the `admin.py` file, so we can modify it in the Django admin section.
+
+```python
+from django.contrib import admin
+from .models import Score
+
+admin.site.register(Score)
+
+```
+Next step is to migrate our model to the database.
+
+Migrations are Django’s way of propagating changes you make to your models (adding a field, deleting a model, etc.) into your database schema.
+
+Now, let's migrate our model to the database by running the commands below:
+
+```bash
+# migrating the app and database changes
+$ python manage.py makemigrations
+
+# final migrations
+$ python manage.py migrate
+```
+
+A superuser has the permissions to create, edit, update, and delete data in Django admin. We create a superuser by running the command below:
+
```

Comment:
Write the second sentence as an instruction for flow. Ie. `Create a superuser by ...`